src/markitdown/_markitdown.py

```diff
@@ -911,6 +915,9 @@ def convert_local(
         ext = kwargs.get("file_extension")
         extensions = [ext] if ext is not None else []
 
+        if not os.path.exists(path):
+            raise FileDoesNotExistException(f"File {path} does not exist")
```

Comment:
Can you place raise the Python built in exception instead?

https://docs.python.org/3/library/exceptions.html#FileNotFoundError